### PR TITLE
refactor: remove tw usages

### PR DESCRIPTION
**Pull request by kt3k**

### CODE REVIEW

File: doc/functions.tsx [hunk 1]
```diff
@@ -232,7 +231,7 @@ export function DocBlockFunction(
             type="radio"
             name={id}
             id={overloadId}
-            class={tw`hidden ${
+            class={`hidden ${
```
Comment (crowlKats) [inline doc/functions.tsx:234]: usages of `css` still require using `tw` if i remember correctly

**Review by crowlKats — COMMENTED** [review]
there are also footer.tsx & icons.tsx in the root of the project

**Comment by kt3k** [comment]
Compared local `_showcase` and https://deno-doc-components.deno.dev/ side by side, and didn't find any visual changes.

**Review by crowlKats — APPROVED** [review]
LGTM!

**Pull request merged**